templates/rabbitmq.config.epp

```diff
@@ -164,7 +171,10 @@
   <%- if $rabbitmq::config::management_ip_address {-%>
       {ip, "<%= $rabbitmq::config::management_ip_address %>"},
   <%- } -%>
-      {port, <%= $rabbitmq::config::management_port %>}
+      {port, <%= $rabbitmq::config::management_port %>}<% if !$rabbitmq::config::config_cowboy_opts.empty {%>,
```

Comment:
Can this match the other one, with a newline after but then suppressing the newline in the template? Or does the open curly brace here need to be on the same line?

I think this will look wrong, but could be wrong.

Probably need to test both cases here, not sure if you'll need to specify any other params to be able to test both the SSL and non-SSL variant.

Reply:
Still looking for a solution to make this more readable. For now this is the only way i found to not get this result :
```
-      {port, 15672},
+      {port, 15672}
+      ,
```

Reply:
got it... I'll take a look. may be possible to suppress the newline entirely with dashes, though I had missed that comma - this may actually be the best / most readable way to do it.

Reply:
I think it's good to leave this as-is. I can make a PR against your branch (side note: ideally, make your PR off a branch other than the default one, i.e., `main` or `master`, in your fork).